addons/event_stand/models/event_stand_type.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class EventStandType(models.Model):
+    _name = 'event.stand.type'
```

Comment:
I would rename to event.stand/booth.category . Purpose is to avoid mixin naming with event.type submodels.